index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<!--[if lte IE 6]><html class="preIE7 preIE8 preIE9"><![endif]-->
+<!--[if IE 7]><html class="preIE8 preIE9"><![endif]-->
+<!--[if IE 8]><html class="preIE9"><![endif]-->
+<!--[if gte IE 9]><!--><html><!--<![endif]-->
+  <head>
+    <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width,initial-scale=1">
+    <title>My ReDi Progject</title>
+  <link rel="shortcut icon" href="favicon.ico" type="image/vnd.microsoft.icon">
+  <link rel="stylesheet" href="stylesheet.css" type="text/css">
+  </head>
+  <body>
+      
```

Comment:
Over all this document has some white space and indentation issues. You could manually go through and fix. No one likes doing that. Auto formatters to the rescue. If you are using VS Code simply edit the file then open the commands menu with Ctrl+Shift+p then type in "Format Document" and execute it with enter key.. 👍

Reply:
Another option that you have with vs code, you can enable automate format when you save your changes.
https://code.visualstudio.com/updates/v1_6#_format-on-save